tap_dynamodb/sync_strategies/full_table.py

```diff
@@ -8,14 +8,17 @@
 LOGGER = singer.get_logger()
 
 
-def scan_table(table_name, projection, last_evaluated_key, config):
+def scan_table(table_name, projection, expression, last_evaluated_key, config):
     scan_params = {
         'TableName': table_name,
         'Limit': 1000
     }
 
     if projection is not None and projection != '':
         scan_params['ProjectionExpression'] = projection
+    if expression:
+        # Add `ExpressionAttributeNames` parameter for reserved word.
+        scan_params['ExpressionAttributeNames'] = json.loads(expression)
```

Comment:
@namrata270998 Add this code block in try-except and handle possible json errors with proper error message.
Add unit test cases for the same